package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-segmentio",
   "description": "The Segmentio analytics.js integration.",
-  "version": "3.2.2",
+  "version": "3.3.0-yolo",
```

Comment:
ooops. Actually I'll avoid changing version in this PR and then change it separately as part of the deploy process